src/config.js

```diff
@@ -122,6 +123,7 @@ export const BZ_QUERIES = {
   },
   unassignedBetaBugs: {
       label: 'Unassigned tracked beta bugs',
+      hidden: 'false',
```

Comment:
can we say that if it isn't define, it is displayed by default?

+ can we use actual boolean instead of string?